diff --git a/dist/index.cjs b/dist/index.cjs
index 0000000000000000000000000000000000000000..1111111111111111111111111111111111111111 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -510,5 +510,5 @@ const BEFORE_AFTER_PSEUDO_RE = /^::?(?:before|after)$/;
 const FONT_FAMILY_RE = /\bfont(?:-family)?\b/i;
-const BEASTIES_COMMENT_RE = /^(?<!! )beasties:(.*)/;
+const BEASTIES_COMMENT_RE = /^(!\s*)?beasties:(.*)/;
 const LEADING_SLASH_RE = /^\//;
 const WHITESPACE_RE = /\s+/;
 const URL_RE = /url\s*\(\s*(['"]?)(.+?)\1\s*\)/;
@@ -875,8 +875,8 @@ class Beasties {
       ast,
       markOnly((rule) => {
         if (rule.type === "comment") {
-          const beastiesComment = rule.text.match(BEASTIES_COMMENT_RE);
-          const command = beastiesComment && beastiesComment[1];
+          const beastiesComment = rule.text.trim().match(BEASTIES_COMMENT_RE);
+          const command = beastiesComment?.[2];
           if (command) {
             switch (command) {
               case "include":
diff --git a/dist/index.mjs b/dist/index.mjs
index 0000000000000000000000000000000000000000..2222222222222222222222222222222222222222 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -500,5 +500,5 @@ const BEFORE_AFTER_PSEUDO_RE = /^::?(?:before|after)$/;
 const FONT_FAMILY_RE = /\bfont(?:-family)?\b/i;
-const BEASTIES_COMMENT_RE = /^(?<!! )beasties:(.*)/;
+const BEASTIES_COMMENT_RE = /^(!\s*)?beasties:(.*)/;
 const LEADING_SLASH_RE = /^\//;
 const WHITESPACE_RE = /\s+/;
 const URL_RE = /url\s*\(\s*(['"]?)(.+?)\1\s*\)/;
@@ -863,8 +863,8 @@ class Beasties {
       ast,
       markOnly((rule) => {
         if (rule.type === "comment") {
-          const beastiesComment = rule.text.match(BEASTIES_COMMENT_RE);
-          const command = beastiesComment && beastiesComment[1];
+          const beastiesComment = rule.text.trim().match(BEASTIES_COMMENT_RE);
+          const command = beastiesComment?.[2];
           if (command) {
             switch (command) {
               case "include":
